dev/docker-compose.yaml

```diff
@@ -22,6 +22,8 @@ services:
       APPLICATION_TITLE: ISACC
       LAUNCH_AFTER_PATIENT_CREATION: "true"
       ENABLE_PROVIDER_FILTER: "true"
+      ENABLE_FILTER_FOR_TEST_PATIENTS: "true"
+      FILTER_FOR_TEST_PATIENTS_LABEL: "Include test recipients"
```

Comment:
i see no value in adding what is now the default for `FILTER_FOR_TEST_PATIENTS_LABEL` to our config.

Reply:
@pbugni the default is `Include test patients` but for ISACC, it should say `Include test recipients`.  That is why I added the config here.

Reply:
i see - thanks for explaining!